eos_data_distribution/avahi_monitor.py

```diff
@@ -184,13 +187,22 @@ def service_removed_cb(self, sda, interface, protocol, name, type, domain, flags
             self.process_route_changes()
 
     def add_nexthop(self, faceUri):
+        if len(self._peers) > MAX_PEERS:
```

Comment:
truedat, added a MIN_PEERS set to 2 so that we don't bother NFD too much too fast.